content/javascript/concepts/variable-scope/variable-scope.md

```diff
@@ -0,0 +1,71 @@
+# Javascript Scope
```

Comment:
This is missing a metadata section, which is a requirement. Refer to the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md) and the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) to see what's required. You can also look at the source of other JavaScript entries to see what we want.

Also, with the metadata in place, we can remove this header. The title will be in the metadata and that's what will populate the entry.